cameo/strain_design/deterministic/flux_variability_based.py

```diff
@@ -344,6 +410,11 @@ def run(self, surface_only=True, improvements_only=True, progress=True, view=Non
         ref_lower_bound = self.reference_flux_ranges.lower_bound.apply(
             lambda v: 0 if abs(v) < non_zero_flux_threshold else v)
```

Comment:
is `ref_lower_bound` obsolete now? Maybe it should be used for `zero_overlap_mask` below?

Reply:
Right, I didn't take care of this.